src/lib/tabwidget/tabcontextmenu.cpp

```diff
@@ -135,6 +136,9 @@ void TabContextMenu::init()
             addAction(tr("Unload Tab"), this, SLOT(unloadTab()));
         }
 
+        addAction(tr("Unload All"), this, SLOT(unloadAllTabsInCurrentWindow()));
+        addAction(tr("Unload All (include Pinned)"), this, SLOT(unloadAllTabsInCurrentWindowWithPinned()));
```

Comment:
This should be after the separator.

The separator separates the actions for **CURRENT** tab and **ALL** tabs

Please send the patch via KDE`s Phabricator as the bot told you.